packages/embark/src/cmd/cmd.js

```diff
@@ -230,8 +230,8 @@ class Cmd {
         embark.simulator({
           port: options.port,
           host: options.host,
-          numAccounts: options.numAccounts,
-          defaultBalance: options.balance,
+          numAccounts: options.accounts,
```

Comment:
This one doesn't look intuitive to me. Shall we at least add a comment to clarify the relation between `accounts` vs `numAccounts` in this case?

Reply:
It's how commander works: `--accounts [numAccounts]`
`accounts` is the actual option, whereas `numAccounts` is just the placeholder